src/main/java/core/basesyntax/workwithfile/filewriter/CsvFileWriter.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.workwithfile.filewriter;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.IOException;
+
+public class CsvFileWriter implements FileWriter {

```

Comment:
Code of this class has nothing specific for csv, it will work fine with txt, xml or anything else
```suggestion
public class FileWriterImpl implements FileWriter {

```